packages/mobile/src/icons/navigator/AccountKey.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import Svg, { Circle, Path } from 'react-native-svg'
+
+export function AccountKey() {
```

Comment:
Does this need to be exported (i.e., do we ever want to use the non-memoized version)?